pkgs/by-name/an/anytype/anytype-heart/default.nix

```
@@ -0,0 +1,89 @@
+{
+  stdenv,
+  lib,
+  callPackage,
+  fetchFromGitHub,
+  buildGoModule,
+  protoc-gen-grpc-web,
+  protoc-gen-js,
+  protobuf,
+}:
+
+let
+  tantivy-go = callPackage ../tantivy-go { };
+  pname = "anytype-heart";
+  version = "0.38.9";
+  src = fetchFromGitHub {
+    owner = "anyproto";
+    repo = pname;
```

Do not specify repo name by the package name

It is considered an antipattern, AFAIK.
```suggestion
    repo = "anytype-heart";
```